Assets/Plugins/Source/Editor/Windows/LogLevelWindow.cs

```diff
@@ -0,0 +1,118 @@
+        }
+        protected override void Setup()
+        {
+            currentLogLevelConfig = EpicOnlineServices.Config.Get<LogLevelConfig>();
+            if (currentLogLevelConfig.logCategoryLevelPairs == null) 
+            {
+                currentLogLevelConfig.logCategoryLevelPairs = new List<LogCategoryLevelPair>();
+            }
+
+            if (currentLogLevelConfig.logCategoryLevelPairs.Count != categories.Length)
+            {
+                currentLogLevelConfig.logCategoryLevelPairs.Clear();
+
+                foreach (var category in categories)
+                {
+                    currentLogLevelConfig.logCategoryLevelPairs.Add(new LogCategoryLevelPair(category, "Info"));
+                }
+
+                SaveToJSONConfig(true);
+            }
+        }
+        protected override void RenderWindow()
+        {
+            // Last element in the list is All Categories
+            previousAllCategoryLogLevel = currentLogLevelConfig.logCategoryLevelPairs[categories.Length - 1].level;
+
+            showAllCategories = EditorGUILayout.Toggle("Show All Log Settings", showAllCategories);
+            EditorGUILayout.LabelField("Category", "Level", GUILayout.Width(200));
+
+            if (showAllCategories)
+            {
+                foreach (var pair in currentLogLevelConfig.logCategoryLevelPairs)
+                {
+                    int selectedLevelIndex = EditorGUILayout.Popup(pair.category, Array.IndexOf(levels, pair.level), levels);
+                    pair.level = LogLevelHelper.LogLevelStringArray[selectedLevelIndex];
+                }
+            }
+            else
+            {
+                EditorGUILayout.BeginHorizontal();
+                selectedCategoryIndex = EditorGUILayout.Popup(selectedCategoryIndex, categories);
+
+                var selectedPair = currentLogLevelConfig.logCategoryLevelPairs[selectedCategoryIndex];
+                int selectedLevelIndex = EditorGUILayout.Popup(Array.IndexOf(levels, selectedPair.level), levels);
+                selectedPair.level = LogLevelHelper.LogLevelStringArray[selectedLevelIndex];           
+                EditorGUILayout.EndHorizontal();
+            }
+
+            // If Level for All Category is "dirty", then update every category
+            var newAllCategoryLogLevel = currentLogLevelConfig.logCategoryLevelPairs[categories.Length - 1].level;
+            if (newAllCategoryLogLevel != previousAllCategoryLogLevel) 
+            {
+                foreach (var pair in currentLogLevelConfig.logCategoryLevelPairs)
+                {
+                    pair.level = newAllCategoryLogLevel;
+                }
+            }
+
+
+
+            if (GUILayout.Button("Save All Changes"))
+            {
+                SaveToJSONConfig(true);
+            }
+        }
+
+        private void SaveToJSONConfig(bool prettyPrint)
+        {
+            currentLogLevelConfig.Write(prettyPrint);
+        }
```

Comment:
It feels like this can be simplified. You can remove the function `SaveToJSONConfig` (functions with that name have been removed from all the other windows that used to utilize it). Subsequent to removing that function, you can replace line `108` with simply `currentLogLevelConfig.Write()` (the `prettyPrint` parameter defaults to `true` if I recall correctly).